### PR TITLE
Expose exception while running test

### PR DESCRIPTION
**Pull request by weiguoz**

### CODE REVIEW

**Review by Yancey0623 — APPROVED** [review]

File: src/test/java/org/sqlflow/client/RunSQLFlowRemoteTest.java [hunk 1]
```diff
@@ -17,6 +17,13 @@ public static void main(String[] args) {
       System.out.println("skip the test due to the SQLFlow client is not created");
       return;
     }
-    EnvironmentSpecificSQLFlowClient.hasGoodResponse(client, sqlProgram);
+
+    client.run(sqlProgram);
+    try {
+      client.release();
+    } catch (InterruptedException e) {
+      System.err.println("encounter an exception while releasing SQLFlow client");
```
Comment (brightcoder01) [inline src/test/java/org/sqlflow/client/RunSQLFlowRemoteTest.java:25]: Print the error message of the exception?

File: src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java [hunk 2]
```diff
@@ -36,6 +37,22 @@ public void runTest() {
       System.out.println("skip remote test");
       return;
     }
-    assert EnvironmentSpecificSQLFlowClient.hasGoodResponse(client, "SELECT 2");
+    try {
+      client.run("SELECT 2");
+    } catch (Exception e) {
+      System.err.println("encounter an exception while running SQLFlow");
```
Comment (brightcoder01) [inline src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java:43]: Same as above.

File: src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java [hunk 3]
```diff
@@ -36,6 +37,22 @@ public void runTest() {
...
+      assert false;
```
Comment (brightcoder01) [inline src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java:44]: We don't need assert false here. Exception will cause the test failure.

File: src/test/java/org/sqlflow/client/RunSQLFlowRemoteTest.java (hunk 1 above)
Comment (weiguoz) [reply src/test/java/org/sqlflow/client/RunSQLFlowRemoteTest.java:25]: Sure. You are right.

File: src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java (hunk 3 above)
Comment (weiguoz) [reply src/test/java/org/sqlflow/client/SQLFlowRemoteTest.java:44]: Good idea.

**Pull request merged**